cmd/kudoctl/cmd/root.go

```diff
@@ -0,0 +1,36 @@
+package cmd
```

Comment:
has this moved?

Reply:
I think `refactoring` in Goland did this quirk when renaming `maestroctl` into `kudoctl`.

I can see it working on my end:
```
$ go run cmd/kudoctl/main.go list instances
List of current instances in namespace "default":
.
```